### PR TITLE
028 minitest

### PR DESCRIPTION
**Pull request by b264**
# no merge

### CODE REVIEW

**Comment by b264** [comment]
has intermittant failures

File: test/legacy/test_en_au_ocker_locale.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 require File.expand_path(File.dirname(__FILE__) + '/../test_helper.rb')
 
-class TesetFakerEnAuLocale < Test::Unit::TestCase
```
Comment (NikitaAvvakumov) [inline test/legacy/test_en_au_ocker_locale.rb:3]: Seriously?! :laughing:

File: test/test_helper.rb [hunk 2]
```diff
@@ -13,3 +11,6 @@
   file.split('.').first
 end
 I18n.enforce_available_locales = true
+
+# configure FactoryHelper
+FactoryHelper::Config.locale = nil
```
Comment (b264) [inline test/test_helper.rb:16]: Not sure if this the best place for this

Comment (NikitaAvvakumov) [inline test/test_helper.rb:16]: Seems good to me

**Pull request closed without merging**